tests/c_languages/CPPCleanBearTest.py

```diff
@@ -17,6 +18,33 @@
 """
 
 
+test_include_paths_file = """
+#include "foo.h"
+#include "bar.h"
+
+int main() {
+    foo();
+    bar();
+    return 0;
+}
+"""
+
+
+def get_testdirectory_name(test_file):
+    return os.path.join(os.path.dirname(__file__), 'test_files', test_file)
+
+
 CPPCleanBearTest = verify_local_bear(CPPCleanBear,
                                      valid_files=(good_file,),
-                                     invalid_files=(bad_file,))
+                                     invalid_files=(bad_file,
+                                                    test_include_paths_file,),
+                                     tempfile_kwargs={'suffix': '.cpp'})
```

Comment:
please move this change in L41 into an own commit ;)

Reply:
So you asked me to move 'test_include_paths_file' to the CPPCleanBearTest. It doesn't work without explicitly having L41. I can either have a different test as I had before for it or I will have to include it here

Reply:
it doesn't work without? okay then nvm :)